gating/update_dependencies/run

```diff
@@ -1,45 +1,39 @@
 #!/bin/bash -xe
 
-## Update OSA SHA to head of stable/ocata
-
+## Update OSA SHA to head of stable/pike
 # These var must be set per branch of RPC-Openstack
-osa_branch=stable/ocata
+osa_branch=stable/pike
 rpco_branch=${BRANCH:-master} # BRANCH injected by Jenkins.
 
 # Env vars injected by Jenkins:
-WORKSPACE="${WORKSPACE:-$PWD}"
+WORKSPACE="${WORKSPACE:-/opt}"
 
 # Note for local testing: this script expects to be executed from the root
 # of an rpco clone, checkout at master.
 
-# Get current head of stable/ocata
-git clone "https://github.com/openstack/openstack-ansible" ${WORKSPACE}/oa
-pushd ${WORKSPACE}/oa
+# Get current head of osa
+osa_dir="${WORKSPACE}/openstack-ansible"
+git clone "https://github.com/openstack/openstack-ansible" ${WORKSPACE}/openstack-ansible
```

Comment:
should `${WORKSPACE}/openstack-ansible` just be `${osa_dir}`?

Reply:
It can be however I've been backing out some of our extra vars. However I'll add this to a follow PR.

Reply:
done.